pkg/ext/apiserver_authorization_test.go

```diff
@@ -311,6 +314,66 @@ func (s *ExtensionAPIServerSuite) TestAuthorization() {
 			},
 			expectedStatusCode: http.StatusForbidden,
 		},
+		{
```

Comment:
I find it a bit confusing that `read-only` means read-only but only for all resources AND for openapi v2, but not v3. I think it would be less confusing to make up new users specifically for those cases. eg: a `openapi-v2-only-read` user and a `openapi-read-only` user, or something similar.

Reply:
Ok, user changed!